### DISCUSSION TITLE
zh/demo

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#443] [Announcements]
# zh/demo

为了让你更好的了解如何使用插件，我们使用 vuepress-plugin-comment2 和 @vuepress/theme-default 制作了一个案例。 源代码 (https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/main/demo/comment2/); 预览 (h...

https://plugin-comment2.vuejs.press/zh/demo.html

**Comment by coderclsld** [comment]
dafsdf

**Comment by gzwrrr** [comment]
fdsfsd

**Comment by Huxzhi** [comment]


**Reply by Mister-Hope** [reply]
不支持

- https://v2.vuepress.vuejs.org/zh/guide/migration.html#%E7%94%A8%E6%88%B7%E9%85%8D%E7%BD%AE%E5%8F%98%E6%9B%B4
- https://v2.vuepress.vuejs.org/zh/guide/plugin.html

**Comment by wuxin0011** [comment]
test

**Comment by wuxin0011** [comment]
test
